cargo-dist/tests/snapshots/axolotlsay_user_plan_job.snap

```diff
@@ -3262,7 +3262,7 @@ on:
 jobs:
   # Run 'cargo dist plan' (or host) to determine what tasks we need to do
   plan:
-    runs-on: ubuntu-latest
```

Comment:
as per previous comment, i think it's better if these remain on ubuntu-latest as i think that is the most common platform users will use